src/util/OAuth2Util.js

```diff
@@ -40,7 +40,7 @@ define(['okta', './Enums', './Errors'], function (Okta, Enums, Errors) {
     _.extend(
       oauthParams,
       _.pick(options, 'clientId', 'redirectUri'),
-      _.pick(options.authParams, 'responseType', 'responseMode', 'display', 'scope', 'scopes'),
```

Comment:
Check out settings.js - I'm now handling all deprecations right when the options come in, and converting them to the supported options. It's nicer because it makes internal code more consistent with the way we want it to look.